Misc/NEWS.d/3.7.0b2.rst

```diff
@@ -357,7 +357,7 @@ Wirtel
 
 Add TLSVersion constants and SSLContext.maximum_version / minimum_version
 attributes. The new API wraps OpenSSL 1.1
-https://www.openssl.org/docs/man1.1.0/ssl/SSL_CTX_set_min_proto_version.html
+https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_min_proto_version.html
```

Comment:
Maybe we should use an archive version here, as this text says it added TLSVersion constants, and 1.1.1 added a `TLS1_3_VERSION` not present in the 1.1.0 archive.

```suggestion
https://web.archive.org/web/20180309043602/https://www.openssl.org/docs/man1.1.0/ssl/SSL_CTX_set_min_proto_version.html
```